staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2215,13 +2215,20 @@ type VirtualMachineInstanceFileSystemList struct {
 	Items           []VirtualMachineInstanceFileSystem `json:"items"`
 }
 
+// VirtualMachineInstanceFileSystemDisk represents the guest os FS disks
+type VirtualMachineInstanceFileSystemDisk struct {
+	Serial  string `json:"serial"`
+	BusType string `json:"bus-type"`
```

Comment:
This does break convention, it should be `json:"busType"`.

Reply:
@xpivarc Thanks for the heads up! this check must be automated somehow, it is something hard to remember for the long term.

Reply:
we can make the `make generate` fail

Reply:
I assumed we wanted to expose with exact naming naming scheme as libvirt

Reply:
Yeah me too. It prevents adding some unnecessary code, but I guess it's odd to only add the exception to BusType.